core/service/storage-service.js

```diff
@@ -59,6 +59,38 @@ var StorageService = exports.StorageService = Montage.specialize({
         }
     },
 
+    getVolumeSnapshotData: {
+        value: function() {
+            return this._dataService.fetchData(Model.VolumeSnapshot).then(function (snapshots) {
+                return snapshots;
+            });
+        }
+    },
+
+    getShareData: {
+        value: function() {
+            return this._dataService.fetchData(Model.Share).then(function (shares) {
+                return shares;
+            });
+        }
+    },
+
+    getPeerData: {
+        value: function() {
+            return this._dataService.fetchData(Model.Peer).then(function (peers) {
+                return peers;
+            });
+        }
+    },
+
+    getReplicationData: {
```

Comment:
This has nothing to do with storage, it should be moved in `replication-service`